### PR TITLE
new companies, typos

### PR DESCRIPTION
**Pull request by dennis-wei**
CORE Circles: added new confirmed companies, fixed typos

Edit: Sorry, added screenshots
screenshot 2016-02-05 20 43 35
screenshot 2016-02-05 20 43 49

Edit2: Fixed link

### CODE REVIEW

**Comment by jackiehluo** [comment]
Screenshot would be great!

File: templates/corecircles.html [hunk 1]
```diff
@@ -49,7 +51,18 @@ <h2 class="apply-now-title">CORE Circles</h2>
 	<a href="https://www.addepar.com/" target="_blank"><img class="gallery-image" src="static/img/circles/addepar.jpg"></a>
 	<p><br>Adeppar is a technology company that supports wealth management companies whose assets under management totals $300 billion.</p>
 
-	<br><br>
+	<br><br><br>
+
+	<a href="https:https://generalassemb.ly/" target="_blank"><img class="gallery-image" src="static/img/circles/generalassembly.png"></a>
```
Comment (jackiehluo) [inline templates/corecircles.html]: That anchor looks off!

**Comment by joshuazweig** [comment]
Is Feb 6 the right date? Thats in three hours

**Comment by dennis-wei** [comment]
Feb 6th 11:59

**Pull request merged**